sklearn/utils/tests/test_utils.py

```diff
@@ -525,13 +525,13 @@ def test_shuffle_dont_convert_to_array():
     a_s, b_s, c_s, d_s, e_s = shuffle(a, b, c, d, e, random_state=0)
 
     assert a_s == ["c", "b", "a"]
-    assert type(a_s) == list
+    assert isinstance(a_s, list)
```

Comment:
Same here? Why do we need this?

Reply:
To pass linting

Reply:
It seems like it was resolved upstream, so I just merged in changes from sklearn:main and reverted these lines for you.

Reply:
The motivation is to keep diff wrt sklearn:main as small as possible cuz it just makes our life easier, so when in doubt, we check upstream to make sure any fixes are already reflected there.